Tests/UnitTests/Shared/Shared.cpp

```diff
@@ -43,6 +43,8 @@ int RunTests(Babylon::Polyfills::Console::CallbackT consoleCallback)
             Napi::Env env = info.Env();
             exitCode.set_value(info[0].As<Napi::Number>().Int32Value());
         }, "SetExitCode"));
+
+        env.Global().Set("RuntimeHostPlatform", Napi::Value::From(env, JSRUNTIMEHOST_PLATFORM));
```

Comment:
```suggestion
        env.Global().Set("hostPlatform", Napi::Value::From(env, JSRUNTIMEHOST_PLATFORM));
```

I also think maybe we should add both `setExitCode` and this to a new object `test` maybe? And use camelCase to be consistent with other JS objects.

Reply:
Adding a new object for a value and a function in this test context looks like over engineering to me. Maybe if it grows (a lot) more. When Unix websocket is available and if test doesn't change, then `hostPlatform` will be removed.